### PR TITLE
Mobile desktop

### PR DESCRIPTION
**Pull request by peterdgreat**
This is a full implementation of the design given, font, icons, colors followed. It was personalized to be an event for a Christian event taking place in Nigeria.
This was built using HTML, CSS, Bootstrap, and Javascript.
All speakers were generated dynamically from Javascript

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,74 @@
...
+- GitHub: [@githubhandle](https://github.com/peterdgreat)
+- Twitter: [@twitterhandle](https://twitter.com/dev_Peter_O)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ajayi-peter-4391ab1b5)
+
+## Scrren Recording
+[check Video](https://www.veed.io/download/58464311-2488-4fa3-8159-66df55a83a3c)
+
+👤 **Credits : Cindy Shin**
+
+-License: [Cindy-Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
```
Comment (AshakaE) [inline README.md]: - [ ] Kindly put this under acknowledgements and not license 🙏

File: README.md [hunk 2]
```diff
@@ -1 +1,74 @@
...
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/peterdgreat)
+- Twitter: [@twitterhandle](https://twitter.com/dev_Peter_O)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ajayi-peter-4391ab1b5)
+
+## Scrren Recording
```
Comment (AshakaE) [inline README.md]: - [ ] Please correct the typo here, should be `screen`  🙂

File: about.html [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+      <div class="navbar-brand d-none d-md-block"
+          >              <img src="img/icons/idea.png" alt="brand"> 
+          <h1>
+            L2 Global <br> Conference 2021
+          </h1></div>
+        <p class="text-center">2021 Illuminations. some right reserved</p>
+      </section>
+      <p class="text-center">  
+        License and Credits to Cindy Shin: https://www.behance.net/adagio07</p>
+    </footer>
```
Comment (AshakaE) [inline about.html]: The footer section is missing a logo on the left

- [ ] Kindly add the logo on the left.
 _see reference below_
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+        <div class="navbar-brand d-none d-md-block"
+        >              <img src="img/icons/idea.png" alt="brand"> 
+        <h1>
+          L2 Global <br> Conference 2021
+        </h1></div>
+          <p class="text-center">2021 Illuminations. some right reserved</p>
+      </section>
+      <p class="text-center">  
+        License and Credits to Cindy Shin: https://www.behance.net/adagio07</p>
+  </footer>
```
Comment (AshakaE) [inline index.html]: The footer section is missing a logo on the left

- [ ] Kindly add the logo on the left.
 _see reference below_
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,327 @@
...
+          <section class="col-md-6 d-flex  f-speaker mb-5">
+            <div class="img">
+              
+            </div>
+            <div class="d-flex flex-column  ps-3">
+              <h3></h3>
+              <p></p>
+              <p></p>
+            </div>
+          </section>
```
Comment (AshakaE) [inline index.html:287]: The articles section has 6 articles, the container for the speaker should be a `section` tag and each speaker item an `article` tag

- [ ] Kindly update this section with the required number of articles
- [ ] Kindly use semantic HTML tags

File: index.html [hunk 6]
```diff
@@ -0,0 +1,327 @@
...
+        </div>
+        <div
+          class="see-height d-flex justify-content-center align-items-center"
+        >
+          <button class="d-lg-none bg-join">
+            Join L2 Ministries Conference 2021
+          </button>
+          <p class="under-p d-none d-lg-block">SEE THE WHOLE PROGRAM</p>
+        </div>
+      </section>
```
Comment (AshakaE) [inline index.html]: - [ ] Kindly make use of adequate proportionality to space elements in this section, i.e icon size, font weight and padding
_see reference below_
image

File: about.html [hunk 7]
```diff
@@ -0,0 +1,278 @@
...
+        </div>
+        <p>
+          The L2 Global Conference Logo was decided through the logo competition
+          from 8.June to 7.July
+        </p>
+        <div class="logo-design">
+          <img src="img/icons/idea.png" alt="brand"> 
+          <h1>
+            L2 Global <br> Conference 2021
+          </h1>
```
Comment (AshakaE) [inline about.html:163]: Only use one `h1` tag per page, every other headings can use a `hx` tag i.e `h2, h3, h4...`

File: about.html [hunk 8]
```diff
@@ -0,0 +1,278 @@
...
+          justify-content-around
+          footer-lg
+          align-items-center
+        "
+      >
+      <div class="navbar-brand d-none d-md-block"
+          >              <img src="img/icons/idea.png" alt="brand"> 
+          <h1>
+            L2 Global <br> Conference 2021
+          </h1></div>
```
Comment (AshakaE) [inline about.html]: Only use one `h1` tag per page, every other headings can use a `hx` tag i.e `h2, h3, h4...`

File: index.html [hunk 9]
```diff
@@ -0,0 +1,327 @@
...
+        <li class="px-2 py-1"><a href="#">English</a></li>
+      </ul>
+    </nav>
+    <nav class="navbar navbar-expand-lg">
+      <div class="container">
+        <a class="navbar-brand d-none d-md-block" href="./index.html"
+          >              <img src="img/icons/idea.png" alt="brand"> 
+          <h1>
+            L2 Global <br> Conference 2021
+          </h1></a
```
Comment (AshakaE) [inline index.html:56]: Only use one `h1` tag per page, every other headings can use a `hx` tag i.e `h2, h3, h4...`

File: index.html [hunk 10]
```diff
@@ -0,0 +1,327 @@
...
+        </div>
+
+        <div class="d-none d-lg-block justify-content-end" id="navbarNav">
+          <div class="d-flex align-items-center justify-content-around">
+            <a class="navbar-brand d-lg-none" href="./index.html"
+              >
+              <img src="img/icons/idea.png" alt="brand"> 
+              <h1>
+                L2 Global <br> Conference 2021
+              </h1>  </a
```
Comment (AshakaE) [inline index.html:69]: Only use one `h1` tag per page, every other headings can use a `hx` tag i.e `h2, h3, h4...`

File: about.html [hunk 11]
```diff
@@ -0,0 +1,278 @@
...
+          <img src="img/icons/square.png" alt="menu" />
+        </div>
+
+        <div class="d-none d-lg-block justify-content-end" id="navbarNav">
+          <div class="d-flex align-items-center justify-content-around">
+            <a class="navbar-brand d-lg-none" href="./index.html"
+              >              <img src="img/icons/idea.png" alt="brand"> 
+              <h1>
+                L2 Global <br> Conference 2021
+              </h1></a
```
Comment (AshakaE) [inline about.html:65]: Only use one `h1` tag per page, every other headings can use a `hx` tag i.e `h2, h3, h4...`

File: index.html [hunk 12]
```diff
@@ -0,0 +1,327 @@
...
+  <div class=" underline">
+      <hr >
+    </div>
+   
+  </div>
+  <ul class="d-flex list-unstyled flex-wrap justify-content-around col-12">
+    <li>Petra Ministries</li>
+    <li>Mustard Seed Unilab </li>
+    <li>TACSFON UI</li>
+  </ul>
```
Comment (AshakaE) [inline index.html:307]: Partner section is missing logos

- [ ] Kindly update this section
_see reference_
image

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @peterdgreat  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (peterdgreat) [reply index.html]: This footer section has a logo already on it, please kindly check this screenshot below
logo

File: about.html (hunk 3 above)
Comment (peterdgreat) [reply about.html]: Kindly check the logo below please
logo

File: index.html (hunk 5 above)
Comment (peterdgreat) [reply index.html:287]: Thanks but I believe we were to make this implementation as personal as possible, not just a dummy copy of template

File: index.html (hunk 12 above)
Comment (peterdgreat) [reply index.html:307]: I believe you were referring to this and which I used the hx tags as logos, This is also here on my page.

I believe you were referring to this

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @peterdgreat  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**